main.cpp

```diff
@@ -79,43 +98,102 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+bool eat(int pizzaSlice, char pizzaName)
+
+{
+  ignoreUnused(pizzaSlice, pizzaName);
+
+  return {};  
+}
 /*
  2)
  */
+void looper(int barSize, double sampleRate = 44.1f)
+
+{
+  ignoreUnused(barSize, sampleRate);
+
+}
 
 /*
  3)
  */
+int speedUp(int distance, int elapsed)
+
+{
+  ignoreUnused(distance, elapsed);
+
+  return {};  
+}
 
 /*
  4)
  */
-
+bool airConditionerStart(int temp, bool smartSaver)
+
+{
+  ignoreUnused(temp, smartSaver);
+
+  return {};  
+}
 /*
  5)
  */
-
+int checkOut(double pricing, float vatAmount)
+
+{
+  ignoreUnused(pricing, vatAmount);
+
+  return {};  
+}
 /*
  6)
  */
-
+double monthlySubscriptionPricing(int month, bool autoRenew)
+
+{
+  ignoreUnused(month, autoRenew);
+
+  return {};  
+}
 /*
  7)
  */
-
+double cardStatement(double dueDate, double minPayAmount)
+
+{
+  ignoreUnused(dueDate, minPayAmount);
+
+  return {};
+}
 /*
  8)
  */
-
+bool setTimer(int hour, int min)
+
+{
+  ignoreUnused(hour, min);
+
+  return{};
+}
 /*
  9)
  */
-
+char surName(int numbOfChar)
+
+{
+  ignoreUnused(numbOfChar);
+
+  return{};
+}
 /*
  10)
  */
-
+void reaVerb(double roomSize = 12.32, bool byPass = false)
+
```

Comment:
This function name does not contain a verb.
Name functions what they do.
- [ ] revise